### PR TITLE
Includes presence of ad blocker in reports (closes #26).

### LINKED ISSUE #26: Reports should include presence of ad-blocker.
This will require the collection of a list of them, and comparison with the list returned here:

https://developer.mozilla.org/en-US/Add-ons/Add-on_Manager/AddonManager#getAllAddons()

It should include the Test Pilot Tracking Protection experiment.

### PR DESCRIPTION
**Pull request by chuckharmston**

### CODE REVIEW

**Comment by chuckharmston** [comment]
Updated to accommodate 7b5e503.

**Review by lmorchard — COMMENTED** [review]
:+1:

File: src/measurements/ad-blocker.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { AddonManager } from 'resource://gre/modules/AddonManager.jsm';
+import BaseMeasurement from '../lib/measurement';
+
+// This list is constructed by manually-reviewing all add-ons in the AMO
```
Comment (lmorchard) [inline src/measurements/ad-blocker.js:4]: Random thought: I wonder if we might be able to solicit additions for any add-ons we're missing via Test Pilot surveys or at least a note in the README in this repo?

Comment (chuckharmston) [reply src/measurements/ad-blocker.js:4]: This isn't a bad thought, but for fairness' sake I trying to come up with a deterministic methodology for inclusion. Pragmatically speaking, this already grabs 99%+ of users, which is probably good enough for these purposes.

**Pull request merged**